.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2723,6 +2723,7 @@ stages:
 
     - script: tracer\build.cmd BuildProfilerSamples BuildAndRunProfilerIntegrationTests --TargetPlatform $(targetPlatform)
       displayName: Run Profiler Integration tests
+      retryCountOnTaskFailure: 2
```

Comment:
I don't think this will work in isolation 🤔 These currently take a long time to run, and doubling the length with likely just cause timeouts...

I think we should probably
- Split out the "build" commands here from the "run" commands
- Add a bunch of retries to those tasks
- Split by target framework as well as target platform (primarily for perf)
- See if we can add features to their custom runner to add the retries
  - If we can't, retry the whole target-framework task: these should be short enough that we _can_ retry the whole lot without hitting timeouts